global-api/migrations/versions/b1e6ab175504_ccra_init_schema.py

```diff
@@ -0,0 +1,88 @@
+"""ccra_init_schema
+
+Revision ID: b1e6ab175504
+Revises: 1d9bbf3aff0e
+Create Date: 2024-10-10 19:45:09.029742
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.sql import text
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'b1e6ab175504'
+down_revision: Union[str, None] = '1d9bbf3aff0e'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+
+    conn.execute(text("""
+        CREATE TABLE IF NOT EXISTS modelled.ccra_indicator (
+          "id" uuid,
+          "actor_id" varchar,
+          "indicator_name" varchar,
+          "indicator_score" numeric,
+          "indicator_units" varchar,
+          "indicator_normalized_score" numeric,
+          "indicator_year" int,
+          "scenario_name" varchar,
+          "datasource" varchar,
+          CONSTRAINT unique_indicator UNIQUE (id)
+        );
+    """))
+
+    conn.execute(text("""
+        CREATE TABLE IF NOT EXISTS modelled.ccra_impactchain (
+          id UUID,
+          keyimpact_name VARCHAR,
+          hazard_name VARCHAR,
+          latest_year INT,
+          scenario_name VARCHAR,
+          CONSTRAINT unique_impactchain UNIQUE (id)
+        );
+    """))
+
+    conn.execute(text("""
+    CREATE TABLE IF NOT EXISTS modelled.ccra_impactchain_indicator (
+      impact_id UUID,
+      indicator_id UUID,
+      actor_id VARCHAR,
+      category VARCHAR,
+      subcategory VARCHAR,
+      indicator_name VARCHAR,
+      indicator_score NUMERIC,
+      indicator_weight NUMERIC,
+      relationship VARCHAR,
+      datasource VARCHAR,
+      CONSTRAINT unique_impactchain_indicator UNIQUE (impact_id, indicator_id, category, subcategory)
+    );
+    """))
+
+    conn.execute(text("""
+    CREATE TABLE IF NOT EXISTS modelled.ccra_riskassessment (
+      "impact_id" uuid,
+      "actor_id" varchar,
```

Comment:
Does this table need its own private key or is the compound key (not fully specified in SQL though) sufficient?
This way there will be no PK index, only a unique index on the table I think.